docs/src/main/asciidoc/datasource.adoc

```diff
@@ -6,58 +6,54 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+Quarkus supports traditional JDBC and modern reactive drivers and allows for unified and flexible configurations.
 
-Both are configured via unified and flexible configuration.
+Quarkus supports lightweight link:https://agroal.github.io/[Agroal] datasource and connection pooling implementation for JDBC, as well as https://vertx.io/[Vert.x] reactive drivers for reactive approaches.
+This provides high performance, scalability, and features integration with the other components in Quarkus,
+such as security, transaction management components, health, and metrics.
 
-[NOTE]
-====
-Agroal is a modern, lightweight connection pool implementation designed for very high performance and scalability,
-and features first class integration with the other components in Quarkus, such as security, transaction management components, health, and metrics.
-====
-
-This guide will explain how to:
-
-* configure a datasource, or multiple datasources
-* how to obtain a reference to those datasources in code
-* which pool tuning configuration properties are available
+For more details about consuming and using a reactive datasource,
+refer to the xref:reactive-sql-clients.adoc[Reactive SQL clients] guide.
 
-This guide is mainly about datasource configuration.
-If you want more details about how to consume and make use of a reactive datasource,
-please refer to the xref:reactive-sql-clients.adoc[Reactive SQL clients guide].
 
-== TL;DR
+== Introduction to datasource configuration
 
-This is a quick introduction to datasource configuration.
-If you want a better understanding of how all this works, this guide has a lot more information in the subsequent paragraphs.
+To start configuring datasources quickly, this section serves as a quick introduction to datasources and provides you with code references you can test and use if you are already familiar with the basics.
+For explainers, examples, and references, refer to the other chapters in this guide.
 
 [[dev-services]]
-=== Zero Config Setup (Dev Services)
+=== Zero-config setup
 
-When testing or running in dev mode Quarkus can even provide you with a zero config database out of the box, a feature
-we refer to as Dev Services. Depending on your database type you may need Docker installed in order to use this feature.
+When testing or running in development mode, Quarkus can provide you with a zero-config database, a feature referred to as _Dev Services_.
 
-If you want to use Dev Services then all you need to do is include the relevant extension for the type of database you want,
-e.g. `jdbc-postgresql`, `reactive-pg-client` or both. Don't configure a database URL, username and password -
-Quarkus will provide the database and you can just start coding without worrying about config.
+Depending on your database type, a container runtime may be installed to use this feature.
+Some databases, such as H2, run in-memory and do not require a container runtime.
```

Comment:
By container runtime, we mean Podman or Docker. This is to say that the database will run inside a container, thus we need a container runtime.

Reply:
Btw in this case "a container runtime may be installed to use this feature" is wrong, since (I hope) Quarkus won't install Podman or Docker by itself.

Reply:
**Needs to be installed** - This is the way :)